packages/location-field/src/index.tsx

```diff
@@ -147,6 +151,157 @@ function makeUserOption(userLocation, index, key, activeIndex, selectHandlers) {
   );
 }
 
+const renderFeature = (
+  itemIndex,
+  layerColorMap,
+  feature,
+  operatorIconMap,
+  setLocation,
+  addLocationSearch,
+  showSecondaryLabels,
+  locationSelectedLookup,
+  activeIndex,
+  GeocodedOptionIconComponent,
+  geocoderConfig
+) => {
+  // generate the friendly labels for this feature
+  const { main, secondary } = generateLabel(feature.properties);
+
+  // Create the selection handler
+  const locationSelected = () => {
```

Comment:
Nit: Careful with creating a new callback with each render (not as bad as creating a new component with each render)